src/wadray_signed.cairo

```diff
@@ -350,7 +350,7 @@ impl SignedWadPartialOrd of PartialOrd<SignedWad> {
 impl SignedRayPartialEq of PartialEq<SignedRay> {
     fn eq(lhs: @SignedRay, rhs: @SignedRay) -> bool {
         let val_cmp: bool = *lhs.val == *rhs.val;
-        if val_cmp && (*lhs.val).is_zero() {
+        if val_cmp && lhs.is_zero() {
```

Comment:
I thought this should be `(*lhs).is_zero()`, but I just learnt that this works too!

Reply:
I think that's because of the difference between `Zero` and `Zeroable`. In the former, the fn is `fn is_zero(self: @T) -> bool` whilst in the latter it's `fn is_zero(self: T) -> bool;`. I haven't noticed until I worked on this PR 😳